README.md

```diff
@@ -1 +1,188 @@
-# Ruby-Catalog-Project
\ No newline at end of file
+<a name="readme-top"></a>
+<div align='center'>
+  <h3><b>**Ruby-Catalog-Project**</b></h3>
+
+</div>
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Ruby-Catalog-Project\] ](#-ruby-catalog-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [Walk-Through Video](#walk-through-video)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Test](#test)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Ruby-Catalog-Project] <a name="about-project"></a>
+In this project, a console app is created that will help you to keep a record of different types of things you own: books, music albums, movies, and games. The data is stored in JSON files 
+and a database with tables structure analogical is also created.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[List all books]**
+- **[List all music albums]**
+- **[List of games]**
+- **[List all genres (e.g 'Comedy', 'Thriller')]**
+- **[List all labels (e.g. 'Gift', 'New')]**
+- **[List all authors (e.g. 'Stephen King')]**
+- **[Add a book]**
+- **[Add a music album]**
+- **[Add a game]**
+- **[All data preserved by saving collections in .json files.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Walk-Through Video -->
+
+## Walk-Through Video <a name="walk-through-video"></a>
+
+[Video](https://drive.google.com/file/d/1893S8dN7oHCV1H5Slgo72zPYZZ1CJS1M/view?usp=drive_link)
+
```

Comment:
![image](https://github.com/zunairkhan811/Ruby-Catalog-Project/assets/93039359/1c51b42d-d9bd-437e-8283-695152180da9)

- [ ] It would be nice if you give us access to the video which you have made 😄 because right now I am only getting this message when I click this link 🤔 Kindly make this video public to make your readme descriptive and professional 👍

Reply:
Access updated